ansible_collections/infoblox/b1ddi_modules/plugins/modules/b1_ipam_subnet.py

```diff
@@ -86,6 +88,7 @@
     tags:
       - {{ key }}: "{{ value }}"
     dhcp_options:
+      - routers: '<first|last|<IP Address>>'
```

Comment:
We should create a sample playbook for the same.

Reply:
Certainly - I have a complex set of playbooks that use all of these features.